exchanges/coinut/coinut_websocket.go

```diff
@@ -384,31 +366,13 @@ func stringToOrderStatus(status string, quantity float64) (order.Status, error)
 }
 
 func (c *COINUT) parseOrderContainer(oContainer *wsOrderContainer) (*order.Detail, error) {
-	var oSide order.Side
-	var oStatus order.Status
-	var err error
-	orderID := strconv.FormatInt(oContainer.OrderID, 10)
-	if oContainer.Side != "" {
-		oSide, err = order.StringToOrderSide(oContainer.Side)
-		if err != nil {
-			c.Websocket.DataHandler <- order.ClassificationError{
-				Exchange: c.Name,
-				OrderID:  orderID,
-				Err:      err,
-			}
-		}
-	} else if oContainer.Order.Side != "" {
-		oSide, err = order.StringToOrderSide(oContainer.Order.Side)
-		if err != nil {
-			c.Websocket.DataHandler <- order.ClassificationError{
-				Exchange: c.Name,
-				OrderID:  orderID,
-				Err:      err,
-			}
-		}
+	side := oContainer.Side
+	if oContainer.Order.Side != 0 {
+		side = oContainer.Order.Side
 	}
 
-	oStatus, err = stringToOrderStatus(oContainer.Reply, oContainer.OpenQuantity)
+	orderID := strconv.FormatInt(oContainer.OrderID, 10)
+	oStatus, err := stringToOrderStatus(oContainer.Reply, oContainer.OpenQuantity)
 	if err != nil {
 		c.Websocket.DataHandler <- order.ClassificationError{
```

Comment:
purge order.ClassificationError type?

Reply:
Yep he means removing it entirely. ATM it's being used inconsistently; not all exchanges use it, we typically return an error on any unmarshal failure for order management and we return errors earlier in the chain for things like getting pairs, pair matching/formatting etc. A lot of work has been put into error wrapping which we can still check for. So the replacement would be to return an error to match everything else, despite liking the original intent of trying to organise order errors. I'm on the side of removal but am open to feedback. An empirical example is that Ryan has a bot doing a tonne of volume and doesn't use it at all

Reply:
I agree. Remove it and replace it with a wrapped error.

Places in go codebase that use struct types for errors drive me crazy, and the core team had a rant about regretting them too.

Reply:
This thread is a personal attack and I am taking my plushie and leaving 4eva

Its an outdated idea I had before we had/before I was aware of lovely wrapping errors

You may kill it along with my hopes and dreams